internal/kaniko/execute.go

```diff
@@ -47,9 +62,123 @@ func (k *Config) Run(ctx context.Context) (err error) {
 			return err
 		}
 	}
+
+	err = k.createArtifactInfo(k.client, k.processDestinations())
```

Comment:
Do we want this to be a blocking call? Are we saying if we cannot publish the artifact info back to the platform the user's workflow should fail even though `kaniko` was able to publish the images?

Reply:
@pinaki124 good question. I have been conflicted about this as well. If we had a way to surface a warning at the workflow run or job level that would be nice. Is there a pattern for logging warnings for users that we could use here?

Reply:
This repo does not use any logging library and you cannot use the private `calculi-corp/log` library since this repo is in a different org, therefore in this case you could simply use the standard go [logging](https://pkg.go.dev/log) library and just do a `log.Printf` and the error can be prefixed with the word `WARN` or something similar.

Reply:
@kushicp In case `createArtifactInfo` returns can error don't return it, simply log the error here with `WARNING: ` prefix and continue.